mod_guestlink_access_form.php

```diff
@@ -0,0 +1,103 @@
+<?php
```

Comment:
This file should likely move into classes/form/guestlink_access_form.php so it gets autoloaded.
See: https://docs.moodle.org/dev/Automatic_class_loading